rust/windows-client/src-tauri/src/client/debug_commands.rs

```diff
@@ -100,59 +99,17 @@ mod details {
     }
 
     pub fn wintun(_: Cli) -> Result<()> {
-        for _ in 0..3 {
-            println!("Creating adapter...");
-            test_wintun_once()?;
-        }
-        Ok(())
-    }
-
-    fn test_wintun_once() -> Result<()> {
-        //Must be run as Administrator because we create network adapters
-        //Load the wintun dll file so that we can call the underlying C functions
-        //Unsafe because we are loading an arbitrary dll file
-        let wintun = unsafe { wintun::load_from_path("./wintun.dll") }?;
-
-        //Try to open an adapter with the name "Demo"
-        let adapter = match wintun::Adapter::open(&wintun, "Demo") {
-            Ok(a) => a,
-            Err(_) => {
-                //If loading failed (most likely it didn't exist), create a new one
-                wintun::Adapter::create(&wintun, "Demo", "Example manor hatch stash", None)?
-            }
-        };
-        //Specify the size of the ring buffer the wintun driver should use.
-        let session = Arc::new(adapter.start_session(wintun::MAX_RING_CAPACITY)?);
-
-        //Get a 20 byte packet from the ring buffer
-        let mut packet = session.allocate_send_packet(20)?;
-        let bytes: &mut [u8] = packet.bytes_mut();
-        //Write IPV4 version and header length
-        bytes[0] = 0x40;
-
-        //Finish writing IP header
-        bytes[9] = 0x69;
-        bytes[10] = 0x04;
-        bytes[11] = 0x20;
-        //...
-
-        //Send the packet to wintun virtual adapter for processing by the system
-        session.send_packet(packet);
-
-        // Sleep for a few seconds in case we want to confirm the adapter shows up in Device Manager.
-        std::thread::sleep(std::time::Duration::from_secs(5));
-
-        //Stop any readers blocking for data on other threads
-        //Only needed when a blocking reader is preventing shutdown Ie. it holds an Arc to the
-        //session, blocking it from being dropped
-        session.shutdown()?;
-
-        //the session is stopped on drop
-        //drop(session);
-
-        //drop(adapter)
-        //And the adapter closes its resources when dropped
+        let subscriber = tracing_subscriber::FmtSubscriber::builder()
+            .with_max_level(tracing::Level::TRACE)
```

Comment:
should we read the `RUST_LOG` env variable for the logs?

Reply:
Yeah. Somehow I lost track of this API - https://docs.rs/tracing-subscriber/latest/tracing_subscriber/fmt/fn.init.html
So I'll just put that in.